app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,17 @@
+class MoviesController < ApplicationController 
+  
+  def show 
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors 
+  end
+
+  def create
```

Comment:
Consider if you have to create a form to add movies. The most reasonable controller action to point that to would be this action (movies#create) but it's being used for setting up the movie-actor relationship. Where else could this go to follow RESTful conventions?